src/headsets/sdk-headset-service.ts

```diff
@@ -93,7 +93,7 @@ export class SdkHeadsetService extends SdkHeadsetBase {
    */
   async endCurrentCall (conversationId: string): Promise<void> {
     if (conversationId) {
-      return this.headsetLibrary.endCall(conversationId);
+      return this.headsetLibrary.endCall(conversationId, false);
```

Comment:
this is no longer needed right?

Reply:
so i believe this is for `hasOtherActiveCalls` but i'm pretty sure we don't... really use that anywhere? i think it defaults to false anyway so not much is different.